### ISSUE TITLE
Path not appling correctly with the install script

### ISSUE THREAD

**Issue by javalsai** [nimsandu/spicetify-bloom#58]
I'm using the linux install script and the patch described in the readme isn't applied correctly, leading to a black screen.

Patch Gotten:
```ini
[Patch]
xpui.js_find_8008 = ,(\w+=)32,
xpui.js_repl_8008 = ,\${1}58,
```

Patch Expected:
```ini
[Patch]
xpui.js_find_8008 = ,(\w+=)32,
xpui.js_repl_8008 = ,${1}58,
```

Result:
![image](https://user-images.githubusercontent.com/80113062/178539983-375d2f37-8ece-43c0-b4ae-1629fa8a0cce.png)

**Comment by javalsai** [comment]
I'll see if I can fix it myself and open a pull request

**Comment by javalsai** [comment]
@nimsandu  Should be solved with #59, it was just a forgetfulness on #53

**Comment by nimsandu** [comment]
@javalsai yah that was even my fault 
im the one who responsible to check before merge
@javalsai  can you work regarding my request on #59
